### ISSUE TITLE
Evaluate of String in the contexts of a Rails 4.x application fails

### ISSUE THREAD

**Issue by tbeauvais** [torquebox/backstage#33]
If you create a simple rails 4.x (4.2.0) application and use Backstage to evaluate against the application pool where the last line is a string (e.g. "test") no results will be displayed. 

There looks to be a compatibility issue in the String from the target application's to_json (from ActiveSupport::JSON), and the JSON generator. Both are using json (1.8.2-java), but when the to_json is called on the resulting String the following exception is thrown. I debugged into the Java code a bit and it looks like RuntimeInfo Java class used by the Generator is not fully populate with all of its context information.

You can workaround this by adding the following to_json method to the String instance you are returning from your Evaluation script.
```
str = "foo"
def str.to_json(options = {})
    "\"#{self}\""
end

# last line of evaluate script
str
```

```
Java::JavaLang::NullPointerException - :
json.ext.Generator$6.generate(Generator.java:385)
json.ext.Generator$6.generate(Generator.java:371)
json.ext.Generator$Handler.generateNew(Generator.java:172)
json.ext.Generator.generateJson(Generator.java:36)
json.ext.GeneratorMethods$RbString.to_json(GeneratorMethods.java:106)
json.ext.GeneratorMethods$RbString$INVOKER$s$0$0$to_json.call(org.jruby.internal.runtime.methods.JavaMethod$JavaMethodN.call(JavaMethod.org.jruby.internal.runtime.methods.DynamicMethod.call(DynamicMethod.org.jruby.internal.runtime.methods.AliasMethod.call(AliasMethod.java:61)
org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:170)
org.jruby.ast.CallOneArgNode.interpret(CallOneArgNode.java:57)
org.jruby.ast.NewlineNode.interpret(NewlineNode.java:105)
org.jruby.ast.IfNode.interpret(IfNode.java:116)
org.jruby.ast.NewlineNode.interpret(NewlineNode.java:105)
org.jruby.evaluator.ASTInterpreter.INTERPRET_METHOD(ASTInterpreter.java:74)
org.jruby.internal.runtime.methods.InterpretedMethod.call(InterpretedMethod.org.jruby.internal.runtime.methods.InterpretedMethod.call(InterpretedMethod.org.jruby.internal.runtime.methods.DefaultMethod.call(DefaultMethod.org.jruby.internal.runtime.methods.AliasMethod.call(AliasMethod.java:76)
org.jruby.RubyClass.finvoke(RubyClass.java:718)
org.jruby.runtime.Helpers.invoke(Helpers.java:512)
org.jruby.RubyBasicObject.callMethod(RubyBasicObject.java:371)
json.ext.Generator$8.generateNew(Generator.java:431)
json.ext.Generator$8.generate(Generator.java:440)
json.ext.Generator$5$1.visit(Generator.java:357)
org.jruby.RubyHash.visitLimited(RubyHash.java:648)
org.jruby.RubyHash.visitAll(RubyHash.java:634)
json.ext.Generator$5.generate(Generator.java:336)
json.ext.Generator$5.generate(Generator.java:309)
json.ext.Generator$Handler.generateNew(Generator.java:172)
json.ext.Generator.generateJson(Generator.java:58)
json.ext.GeneratorState.generate(GeneratorState.java:210)
json.ext.GeneratorState$INVOKER$i$1$0$generate.call(org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:170)
org.jruby.ast.CallOneArgNode.interpret(CallOneArgNode.java:57)
org.jruby.ast.NewlineNode.interpret(NewlineNode.java:105)
org.jruby.ast.BlockNode.interpret(BlockNode.java:71)
org.jruby.evaluator.ASTInterpreter.INTERPRET_METHOD(ASTInterpreter.java:74)
org.jruby.internal.runtime.methods.InterpretedMethod.call(InterpretedMethod.
```

**Comment by tbeauvais** [comment]
I created a pull request for this: https://github.com/torquebox/backstage/pull/34

**Comment by tobias** [comment]
@tbeauvais - thanks for the fix! I've merged it and released backstage 1.1.1 (https://rubygems.org/gems/torquebox-backstage/versions/1.1.1).
